include/glow/Graph/Nodes.h

```diff
@@ -108,21 +108,35 @@ class Constant : public Storage {
   bool verify() const;
 };
 
+uint32_t getNewPlaceholderId();
```

Comment:
Why is this a part of the Node? I would expect the mapping to be done by the Module/context or some other centralized place.